src/utils/json.ts

```diff
@@ -46,5 +46,5 @@ export function getPropertyFromJsonString(
   propertyName: string
 ): any {
   const json = tryParseJson(jsonString);
-  return (json && json[propertyName]) || null; // TODO: Handle falsy values.
+  return (json && json[propertyName]) ?? null;
```

Comment:
The updated code returns the value of the property, or null if the input string is not valid JSON or the given property is not present in the JSON. If that's the desired behavior it should be specifically documented. It might make more sense to write this as

```
return json?.[propertyName];
```

so the method returns `undefined` if the property is missing or the string is not valid JSON, and only returns `null` if the value is the JSON `null` literal.

Reply:
Either way the behavior should be verified with unit tests.

Reply:
Good call, I updated the logic and added a test